src/Messages/PushMessage.php

```diff
@@ -151,6 +168,19 @@ public function badge($badge)
         return $this;
     }
 
+    /**
+     * Set notification icon color.
+     *
+     * @param string $color
+     * @return $this
+     */
+    public function color($color = '')
```

Comment:
Maybe we could remove the default argument value since I'm not sure if the empty string will be treated as a valid color in the app which receives the notification.